### PR TITLE
Update content location by transfer and add information

### LINKED ISSUE #187: Format is not only allowed for string values
At https://json-schema.org/understanding-json-schema/reference/string.html#format it says:
> The format keyword allows for basic semantic validation on certain kinds of **string** values that are commonly used. 

As discussed in issue https://github.com/json-schema-org/json-schema-spec/issues/759, format is also defined for non-string types. Therefore, the description should be made more general and moved to another place as it doesn't really fit where it is now. I guess it's better placed at the end of https://json-schema.org/understanding-json-schema/reference/type.html or so...

### PR DESCRIPTION
**Pull request by jagpreetrahi**
**What kind of change does this PR introduce?**

  This PR allows content to be updates and reorganizes the content of the Markdown files used in the JSON website. 

**Issue Number:**
 Pick one of the below options.  Please remove those which don't apply. 
-  Closes #187  <!-- Replace 187 with the issue number this PR resolves 

**Screenshots/videos:**

Screenshot (72)
Screenshot (73)
Screenshot (74)

**If relevant, did you update the documentation?**

Yes, the documentation has been updated to reflect the changes made in the Markdown files.  
    [Local  Server](http://localhost:3000/understanding-json-schema/reference/type) 

**Summary**

 This change focuses on updating and moving content between Markdown files to enhance the clarity and relevance of information presented on the website. The motivation behind these updates is to provide a more cohesive user experience by reorganizing related content into more logical sections. 
 [Issue - 187](https://github.com/json-schema-org/website/issues/187) 

**Does this PR introduce a breaking change?**

 No, this PR does not introduce any breaking changes. The website's structure and functionality remain intact, ensuring a smooth user experience.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,3 +60,7 @@ yarn-error.log*
 !.yarn/sdks
 !.yarn/versions
 
+#local file
+.env
+/node_modules
+
```
Comment (JeelRajodiya) [inline .gitignore:66]: Please remove these changes, we already have .env and /node_modules in the file

Comment (jagpreetrahi) [reply .gitignore:66]: ok

Comment (JeelRajodiya) [inline .gitignore:66]: Please remove these changes, we already have .env and /node_modules in this file

```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -60,3 +60,4 @@ yarn-error.log*
 !.yarn/sdks
 !.yarn/versions
 
+
```
Comment (JeelRajodiya) [inline .gitignore:63]:
```suggestion
```
Remove this extra line as well

**Comment by DhairyaMajmudar** [comment]
@jagpreetrahi Thank you for your contribution but it seems like there's PR #963  is performing the same things done in this one. Can you pls. tell if anything is out of scope for improvement done in this, if not can you pls. mind closing it

**Comment by jagpreetrahi** [comment]
Hi @DhairyaMajmudar, thank you for your response! In this issue, I aimed to provide additional details and examples, especially for non-string types, that are not currently mentioned in the official documentation. I also compared it with PR #963, and while the intention is similar, my contribution includes extra relevant information that could be helpful for developers. If these changes are useful, feel free to merge them; otherwise, I'm happy to close the issue. Please don't hesitate to reach out if any other issues arise that I can help with—I’d love to continue contributing to this organization!

**Comment by benjagm** [comment]
Hi @jagpreetrahi . Please apologize me as I am responsible of 2 contributors working on the same issue by mistake. You did a great work, however most of the discussion is happening in #963 therefore we are going to focus on that one.

I'd like to thank you for your time and efforts and we'd like you to join that discussion as PR reviewer therefore all the knowledge and experience you gained with this contribution will be used to make sure the other PR is great.

**Comment by jagpreetrahi** [comment]
Hi @benjagm, hope you’re doing well ! No need to worry about any of it— I appreciate the opportunity to review and discuss #963 further. I’m working with the team to make improvements to the docs for our organization, aiming to get it done as soon as possible. Let’s keep collaborating to make this even better

**Pull request closed without merging**